### PR TITLE
Bug 1227796 - re-enable screen_lock_settings_test.js

### PR DESCRIPTION
**Pull request by gasolin**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=426eb43e0533d805d52638e0182c7f13074da1dd

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0055d215bd7a0f9c65f8d2751194d252b340e1e6

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=be33e5cd75393bf5cc92033d9d6146f92b47b1e8

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a15e7349c2a18f342774606eb6a4a2c4af59e220

File: apps/settings/test/marionette/tests/screen_lock_settings_test.js [hunk 1]
```diff
@@ -98,8 +97,7 @@ marionette('manipulate screenLock settings', function() {
       'passcode is not checked');
   });
 
-  test(
-    'passcode is enabled and won\'t get disabled if you tap back button ' +
+  test('passcode is enabled and won\'t get disabled if you tap back button ' +
     'when we try to disable passcode directly',
     function() {
```
Comment (scottwu) [inline apps/settings/test/marionette/tests/screen_lock_settings_test.js]: This test is giving me intermittent error on my local machine. Not sure if you are experiencing it too? It seems like a slight time delay before the second togglePasscodeLock() on line 109 would fix the problem.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f5b7b6aba9883fedbf25ec51d89a7efe1dbab6bd

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=025c47bef4f709a53e3ce5ad27e676288c0d21f3

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=126b696c896e08f5f26889200a15abd8214a556d

**Pull request merged**